tests/unit/s3/test_transfer.py

```diff
@@ -391,6 +391,18 @@ def test_multipart_download_with_multiple_parts(self):
                           mock.call(Range='bytes=4-7', **extra),
                           mock.call(Range='bytes=8-', **extra)])
 
+    def test_multipart_download_with_multiple_parts_and_extra_args(self):
+        client = mock.Mock()
+        response_body = b'foobarbaz'
+        client.get_object.return_value = {'Body': six.BytesIO(response_body)}
+        downloader = MultipartDownloader(
+            client, TransferConfig(), InMemoryOSLayer({}), SequentialExecutor)
+        extra_args = {'RequestPayer': 'requester'}
+        downloader.download_file(
+            'bucket', 'key', 'filename', len(response_body), extra_args)
+        for call_args in client.get_object.call_args_list:
+            self.assertEqual(call_args[1].get('RequestPayer'), 'requester')
```

Comment:
Oh let's use the stubber for these and the ``expected_params``. I have been using it lately. It's pretty handy and makes the test case much cleaner.